===================================================================
RCS file: /cvsroot/grub/cvsroot/grub/grub/stage2/char_io.c,v
retrieving revision 1.51
retrieving revision 1.52
diff -u -r1.51 -r1.52
--- grub/stage2/char_io.c	2002/12/09 06:27:05	1.51
+++ grub/stage2/char_io.c	2004/03/27 17:02:54	1.52
@@ -1265,3 +1265,9 @@
   return dest;
 }
 #endif /* ! STAGE1_5 */
+
+#ifndef GRUB_UTIL
+# undef memcpy
+/* GCC emits references to memcpy() for struct copies etc.  */
+void *memcpy (void *dest, const void *src, int n)  __attribute__ ((alias ("grub_memmove")));
+#endif
